packages/jaeger-ui/src/utils/date.tsx

```diff
@@ -158,12 +162,12 @@ export const getSuitableTimeUnit = (microseconds: number): string => {
     return 'microseconds';
   }
 
-  const duration = moment.duration(microseconds / 1000, 'ms');
+  const duration = dayjs.duration(microseconds / 1000, 'ms');
 
   return Object.keys(timeUnitToShortTermMapper)
     .reverse()
```

Comment:
`timeUnitToShortTermMapper` keys are just unit names, why is `reverse` meaningful for them? I expect the order of object keys not to be guaranteed in JavaScript. If we need strong order then it should be an array, not a dict.

Reply:
Yes, even I got confused due to this, I'll see which approach is better